modules.d/45simpledrm/module-setup.sh

```diff
@@ -8,17 +8,17 @@ check() {
 # called by dracut
 installkernel() {
     # Include simple DRM driver
-    instmods simpledrm
+    hostonly='' instmods simpledrm
 
-    if [[ $hostonly ]]; then
+    if [[ $hostonly_mode != "strict" ]]; then
```

Comment:
I think the `!=` is supposed to be an `=` here. You want to restrict the modules in strict mode, but not in sloppy mode and definitely not in `hostonly=no` mode. Right now this will run even when `hostonly=no` is set, if I'm understanding correctly.